src/patterns/components/tabs/base.hbs

```diff
@@ -3,10 +3,13 @@ order: 1
 hasAngularCode: true
 hasAngularDocs: true
 hasAngularCodeInfo: true
+hasReactCode: true
+hasReactDocs: true
+hasReactCodeInfo: true
 ---
 
-<div class="sprk-c-Tabs" role="tablist" aria-orientation="horizontal" data-sprk-navigation="tabs" data-id="tabs-1">
-  <div class="sprk-c-Tabs__buttons">
+<div class="sprk-c-Tabs" aria-orientation="horizontal" data-sprk-navigation="tabs" data-id="tabs-1">
+  <div class="sprk-c-Tabs__buttons" role="tablist">
```

Comment:
This fixes a bug in Vanilla Spark Core where the role was placed on the incorrect element. Steph is making a separate issue to track this but it's been decided that it can be merged as part of this PR.

Reply:
Updated!